docs/src/tutorials/exporting.md

```diff
@@ -1,51 +1,176 @@
 ---
 title: "Exporting data"
-description: |
-  Platform.sh aims to be a great host, but we never want to lock you in to our service. Your code and your data belong to you, and you should always be able to download your site's data for local development, backup, or to "take your data elsewhere".
+description: See how to export your code, files and service's data.
 ---
 
-{{% description %}}
+Platform.sh never wants to lock you in to our service.
+Your code and your data belong to you,
+and you should always be able to download your site's data for local development,
+backup, or to "take your data elsewhere".
```

Comment:
🛠️ A suggestion to make the introduction shorter, more user-focused/straightforward and get rid of the conditional ("should" => https://developers.google.com/style/word-list#letter-s). What do you think?

```suggestion
As a Platform.sh user, your code and data belong to you.
At any time, you can download your site's data for local development,
backup, or to change provider.
```

Reply:
Just realized I should have removed the last comma (before "or") in my suggestion.

Reply:
Updated suggestion following conversation with Aaron (the comma does have its place here, but the syntax can be improved).